packages/typespec-vscode/package.json

```diff
@@ -268,18 +268,23 @@
     "package-vsix": "vsce package",
     "deploy": "vsce publish",
     "open-in-browser": "vscode-test-web --extensionDevelopmentPath=. .",
-    "test:e2e": "pnpm test:web",
-    "test:web": "vscode-test-web --quality stable --extensionDevelopmentPath=. --headless --extensionTestsPath=dist/test/web/suite.js ./test/web/data"
+    "test:e2e": "pnpm test:web && pnpm test:extension",
+    "test:web": "vscode-test-web --quality stable --extensionDevelopmentPath=. --headless --extensionTestsPath=dist/test/web/suite.js ./test/web/data",
+    "test:extension": "vitest --root test/extension"
   },
   "devDependencies": {
+    "@nut-tree-fork/nut-js": "^4.2.6",
     "@types/cross-spawn": "~6.0.6",
     "@types/mocha": "^10.0.9",
     "@types/node": "~24.0.3",
     "@types/semver": "^7.5.8",
     "@types/vscode": "~1.100.0",
     "@types/which": "^3.0.4",
     "@typespec/compiler": "workspace:^",
+    "@typespec/http": "workspace:^",
```

Comment:
why do you need http and openapi dependeencies in the test?

Reply:
The `http` and `openapi` packages have been removed. These two packages were dependencies required for the UI cases "Emit TypeSpec Project" and "Import TypeSpec from OpenAPI3". However, in the current PR, they are temporarily not needed for the "Create TypeSpec Project" demo.